warrior/Actions/SnmpActions/common_snmp_actions.py

```diff
@@ -630,75 +669,40 @@ def verify_snmp_action(self, system_name, snmp_result, mib_string=None
                                                    errindex and
                                                    varBindTable[-1][int(errindex)-1][0]or '?'))
             else:
-                if type(varBindTable[0]) is not list:
-                    # for SNMP Get/Get-Next output only
-                    for name, val in varBindTable:
-                        if ws.checkoctetstring(val):
-                            if '0x' == val.prettyPrint()[:2]: # to
-                                # confirm if it is a hex string or not
-                                if len(val.prettyPrint()) == 14:# for MAC
-                                    s = val.prettyPrint()
-                                    val = ':'.join( s[x:x+2] for x in
-                                                    range(2,len(s), 2))
-                                if len(val.prettyPrint()) == 12:# for IP
-                                    s = val.prettyPrint()
-                                    val = '.'.join( s[x:x+2] for x in
-                                                    range(2,len(s), 2))
-
-                            result_list.append((name.prettyPrint(),
-                                                val))
-                            testcase_Utils.pNote('%s = %s' %(
-                                name.prettyPrint(),val))
-                        else:
-                            result_list.append((name.prettyPrint(),
-                                                val.prettyPrint()))
-                            testcase_Utils.pNote('%s = %s' % (
-                                name.prettyPrint(),val.prettyPrint()))
+                if varBindTable:
+                    if type(varBindTable[0]) is not list:
```

Comment:
use isinstance instead of type.

Reply:
type does what this statement intended for. Instance wont be used here.